src/transforms/groupby.js

```diff
@@ -35,14 +35,24 @@ exports.attributes = {
         ].join(' ')
     },
     style: {
-        valType: 'any',
-        dflt: {},
-        description: [
-            'Sets each group style.',
-            'For example, with `groups` set to *[\'a\', \'b\', \'a\', \'b\']*',
-            'and `style` set to *{ a: { marker: { color: \'red\' } }}',
-            'marker points in group *\'a\'* will be drawn in red.'
-        ].join(' ')
+        target: {
+            valType: 'any',
```

Comment:
Not really a problem in JSON, but a little weird in plain JS API usage:

<img width="183" alt="screen shot 2017-03-10 at 13 54 48" src="https://cloud.githubusercontent.com/assets/572717/23808805/37a46a28-0599-11e7-9ae5-36b512bc1bda.png">